src/main/java/uk/gov/justice/hmpps/prison/api/resource/BookingResource.java

```diff
@@ -924,6 +924,7 @@ public CaseNoteCount getCaseNoteCount(@PathVariable("bookingId") @Parameter(desc
         @ApiResponse(responseCode = "500", description = "Unrecoverable error occurred whilst processing request.", content = {@Content(mediaType = "application/json", schema = @Schema(implementation = ErrorResponse.class))})})
     @Operation(summary = "Offender adjudications summary (awards and sanctions).", description = "Offender adjudications (awards and sanctions).")
     @GetMapping("/{bookingId}/adjudications")
+    @VerifyBookingAccess(overrideRoles = {"SYSTEM_USER", "GLOBAL_SEARCH", "VIEW_PRISONER_DATA", "VIEW_ADJUDICATIONS"})
```

Comment:
Moved from `AdjudicationService.getAdjudicationSummary` and added `VIEW_ADJUDICATIONS` - this was the only place calling it so seemed ok to move